plenum/common/stacks.py

```diff
@@ -59,13 +60,17 @@ def transmitToClient(self, msg: Any, remoteName: str):
                 remoteName = remoteName.encode()
             self.send(payload, remoteName)
         except Exception as ex:
-            # TODO: This should not be an error since the client might not have
+            # use warning for PlenumTransportError
+            # one expected case: unknown identity since the client might not have
             # sent the request to all nodes but only some nodes and other
             # nodes might have got this request through PROPAGATE and thus
             # might not have connection with the client.
-            logger.error(
-                "{}{} unable to send message {} to client {}; Exception: {}" .format(
-                    CONNECTION_PREFIX, self, msg, remoteName, ex.__repr__()))
+            # TODO make more accurate for other cases
```

Comment:
According to the comment above, can we consider all exceptions as warnings?

Reply:
It would be too generic. I think we should use warning only for cases that are not desired but expected like described in comment. I'm not sure about all other cases which might be related to some real bug. Moreover usage of warning instead of error for base transport error PlenumTransportError seems too generic as well since the comment relates only to errors like unknown identity. I did that in such way to make old tests pass since in legacy logic no transport exceptions came to the current code at all.